campaign-launcher/client/src/components/CampaignInfo/index.tsx

```diff
@@ -1,19 +1,23 @@
 import { FC } from 'react';
 
-import { Box, Tooltip, Typography } from '@mui/material';
+import { Box, Typography } from '@mui/material';
 
 import { CalendarIcon } from '../../icons';
 import { CampaignDetails } from '../../types';
 import { getChainIcon, getNetworkName, mapStatusToColor } from '../../utils';
-import ExplorerLink from '../ExplorerLink';
-import JoinCampaign from '../JoinCampaign';
+import dayjs from '../../utils/dayjs';
+import CustomTooltip from '../CustomTooltip';
 
 const formatDate = (dateString: string): string => {
-  const date = new Date(dateString);
-  const day = date.getDate();
-  const month = date.toLocaleString('en-US', { month: 'short' });
-  const year = date.getFullYear();
-  return `${day} ${month} ${year}`;
+  return dayjs(dateString).format('D MMM YYYY');
+};
+
+const formatTime = (dateString: string): string => {
+  const date = dayjs(dateString);
+  const offset = date.utcOffset();
+  const sign = offset >= 0 ? '+' : '-';
+  const hours = Math.floor(Math.abs(offset) / 60);
+  return `${date.format('HH:mm')} GMT${sign}${hours}`;
 };
```

Comment:
```suggestion
const formatTime = (dateString: string): string => {
  const date = dayjs(dateString);
  return date.format('HH:mm z');
};
```
but have to add [timezone plugin](https://day.js.org/docs/en/plugin/timezone)
or
```
const formatTime = (dateString: string): string => {
  const date = dayjs(dateString);
  return date.format('HH:mm [GMT]Z');
};
```
if you don't want to use it, but probably with tz plugin it is more user-friendly and it's not that heavy to add